KarhooSDK/Api/Response/Quote.swift

```diff
@@ -10,7 +10,6 @@ import Foundation
 
 public struct Quote: KarhooCodableModel, Equatable {
 
-    public let vehicleAttributes: VehicleAttributes
```

Comment:
Just so you know, this may require some changes on the UISDK as well. Chances are that it uses VehicleAttributes in some way at the moment.

Reply:
done